AccountingService/Controllers/UsersController.cs

```diff
@@ -15,130 +16,155 @@ public class UsersController : ControllerBase
 
         private NetSalaryService netSalaryService;
 
+
+
         public UsersController(SalaryConversionContext context) : base()
         {
             salaryConversionContext = context;
             exchangeRateService = new ExchangeRateService();
             netSalaryService = new NetSalaryService();
+
+
         }
 
         [HttpGet]
         public IActionResult GetAllUsers()
         {
-            var users = salaryConversionContext.User.ToArray(); // izvuci sve user iz tabele(user) 
-            return Ok(new
+            try
             {
-                Count = users.Length,
-                Users = users,
-            });
-
+                var users = salaryConversionContext.User.ToArray(); // izvuci sve user iz tabele(user) 
+                return Ok(new
+                {
+                    Count = users.Length,
+                    Users = users,
+                });
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Error: " + ex.Message);
+                return new StatusCodeResult(500);
+            }
         }
 
-
         [HttpGet("{id}")]
         public IActionResult GetUserById(int id)
         {
-            var user = salaryConversionContext.User.Where(u => u.ID == id).SingleOrDefault();
-            if (user == null)
+            try
             {
-                return NotFound();
-            }
+                var user = salaryConversionContext.User.Where(u => u.ID == id).SingleOrDefault();
 
-            return Ok(user);
 
+                return Ok(user);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Error: " + ex.Message);
```

Comment:
Same as previous comment. Remove this. It is unnecessary.

Reply:
done